src/foam/dao/Relationship.js

```diff
@@ -46,6 +46,7 @@ foam.CLASS({
     },
     {
       name: 'name',
+      class: 'foam.core.String',
```

Comment:
Just use 'String', defaults to 'foam.core.' package.

Reply:
Done.